package.json

```diff
@@ -2,18 +2,18 @@
   "name": "cachet",
   "devDependencies": {
     "gulp": "~3.9.1",
-    "laravel-elixir": "~5.0.0",
+    "laravel-elixir": "~6.0.0-9",
     "animate-sass": "git+https://github.com/tgdev/animate-sass.git",
     "autosize": "^3.0.15",
     "bootstrap-sass": "^3.3.6",
     "chart.js": "^2.1.2",
-    "eonasdan-bootstrap-datetimepicker": "~3.1",
+    "eonasdan-bootstrap-datetimepicker": "~4.15",
```

Comment:
We don't use v4 of the bootstrap datetimepicker yet, because of some weird positioning issues. Could you revert this change please?